controllers/job.go

```diff
@@ -19,6 +19,7 @@ const (
 	jobsPath            = consumerPath + "/queued-jobs"
 	jobIDPathParamKey   = "jobId"
 	jobPath             = consumerPath + "/job/:" + jobIDPathParamKey
+	jobRequeuePath      = consumerPath + "/job/:" + jobIDPathParamKey + "/requeue-dead-job"
```

Comment:
just a note... `requeue-dead-job` sounds kinda unnecessarily verbose. maybe `requeue` should convey the same message?

Reply:
I think `requeue` is misleading to be honest since not every job can be requeued; hence this name. I prefer verbose names since ideally no will have to "craft" the URLs but rather discover them; for example requeue URLs are discoverable in the single job get endpoint and the DLQ endpoint.